shortfin/python/shortfin_apps/llm/k8s/llama-app-deployment.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
```

Comment:
Should this be included in the same dir as the source code, or would it make sense to have it in its own location?

i.e. `deployment/shortfin_apps/llm/k8s/llama-app-deployment.yaml`

Also some pre-commit errors on the structure of the file:

```
expected a single document in the stream
  in "shortfin/python/shortfin_apps/llm/k8s/llama-app-deployment.yaml", line 1, column 1
but found another document
  in "shortfin/python/shortfin_apps/llm/k8s/llama-app-deployment.yaml", line 46, column 1
  ```

Reply:
True, moved over to its own space and updated check-yaml hook